### ISSUE TITLE
Migrate `Demo` to coroutines storage

### ISSUE THREAD

**Issue by nulls** [saveourtool/save-cloud#1898]

**Comment by sanyavertolet** [comment]
@nulls is this issue still actual?

**Comment by nulls** [comment]
Yes, but I only added implementation for coroutines, but didn't migrate existed storage to it. 
It requires a big refactoring in demo project

**Comment by akuleshov7** [comment]
> Yes, but I only added implementation for coroutines, but didn't migrate existed storage to it.
> It requires a big refactoring in demo project

very funny
